packages/flutter/lib/src/scheduler/binding.dart

```diff
@@ -183,6 +183,34 @@ enum SchedulerPhase {
   postFrameCallbacks,
 }
 
+/// This callback is invoked when a request for [DartPerformanceMode] is disposed.
+///
+/// See also:
+///
+/// * [PerformanceModeRequestHandle] for more information on the lifecycle of the handle.
+typedef PerformanceModeCleaupCallback = void Function();
```

Comment:
Since this type isn't used anywhere, it can be a `VoidCallback` instead

Reply:
It is still used in `PerformanceModeRequestHandle`, I think this typedef makes it easier to see when this callback will be invoked.

Reply:
Sorry, I meant that the type is not used in a public APi, so this can be private or use an existing type - right?

Reply:
Ah right, will change it to private.